Source/Core/Core/CoreTiming.h

```diff
@@ -31,6 +31,7 @@ extern u64 g_fake_TB_start_ticks;
 extern int g_slice_length;
 extern float g_last_OC_factor_inverted;
 
+// NOTE: ScheduleEvent will not start updating the PowerPC downcount until after the first Advance.
```

Comment:
The actual limitation is: For things to work as expected, `Advance()` must be called before the first cycle is executed.